src/AddTodoForm.jsx

```diff
@@ -15,18 +16,18 @@ function AddTodoForm({ onAddTodo }) {
     setTodoTitle("");
   };
 
+  const focus = () => inputRef.current.focus();
+
   return (
     <form onSubmit={handleAddTodo}>
-      <label htmlFor="todoTitle">Title</label>
-      <input
-        id="todoTitle"
-        name="title"
-        value={todoTitle}
-        onChange={handleTitleChange}
-      ></input>
-      <button>Add</button>
+      <InputWithLabel
+        todoTitle={todoTitle}
+        onTitleChange={handleTitleChange}
+        label="Title"
```

Comment:
Instead of passing in the label as the prop, the assignment asks to pass this through as a child element as an example of component composition. E.g., `<InputWithLabel>Label</InputWithLabel>`